sonic_platform_base/watchdog_base.py

```diff
@@ -0,0 +1,50 @@
+#
+# watchdog_base.py
+#
+# Abstract base class for implementing a platform-specific class with which
+# to interact with a hardware watchdog module in SONiC
+#
+
+from . import device_base
+
+
+class WatchdogBase(device_base.DeviceBase):
+    """
+    Abstract base class for interfacing with a hardware watchdog module
+    """
+
+    # Possible fan directions
+    FAN_DIRECTION_INTAKE = "intake"
+    FAN_DIRECTION_EXHAUST = "exhaust"
```

Comment:
why fan direction in watchdog base class?

Reply:
Leftover after copy/paste; meant to be deleted. Resolved in d605896.